javascript-functions/main.js

```diff
@@ -0,0 +1,29 @@
+function convertMinutesToSeconds(minutes) {
+  return minutes * 60;
+}
+convertMinutesToSeconds(5);
```

Comment:
if you saved the result in a variable, you could just pass the variable to the console log